### DISCUSSION TITLE
Set a custom unit for unitless properties

### DISCUSSION THREAD

**Discussion by undrscrdt** [vanilla-extract-css/vanilla-extract#978] [Ideas]
I rarely write styles with `px` in mind, I usually want to use `r`/`em` units for accessibility reasons, and `px` is only ever an override to keep a dimension consistent regardless of font size (for example horizontal padding on text).

With how rarely I use `px`, the unitless properties feature is pointless for me, but I would totally use it if `rem` was postfixed instead!

Maybe something like this for the sprinkles package:
```ts
import {defineProperties, createSprinkles } from '@vanilla-extract/sprinkles';

const properties = defineProperties({
    defaultUnits: "rem"
});

export const sprinkles = createSprinkles(properties);
export type Sprinkles = Parameters<typeof sprinkles>[0];
```

**Comment by nksfrank** [comment]
I would love to also be able to customise the unitless unit somehow.
